ansible/library/port_alias.py

```diff
@@ -94,6 +94,11 @@ def get_portmap(self, asic_id=None):
         portmap = {}
         aliasmap = {}
         portspeed = {}
+        # Front end interface asic names
+        asicifnames = []
```

Comment:
rename to asic_ifnames

Reply:
Renamed to front_panel_asic_names. This will be the list of asic names of front panel interfaces.